controllers/productController.js

```diff
@@ -0,0 +1,23 @@
+const productService = require("../services/productService");
+
+const getProducts = async (req, res) => {
+  const { category, sub, type, sort, offset = 1 } = req.query;
```

Comment:
- offset을 1로 설정했을 때 첫번째 row 데이터부터 나오는 지 확인해주세요~
- category와 sub라는 key(변수)가 무엇을 가리키는 지 조금 애매한 것 같습니다. 명확하게 작성해주세요!
```suggestion
  const { categoryId, subCategoryId, type, sort, offset = 1 } = req.query;
```